reactor-core/build.gradle

```diff
@@ -38,7 +38,7 @@ ext {
 		"Export-Package": [
 			"!*internal*",
 			"!reactor.blockhound*",
-			"reactor.*"
+			"reactor.*;version=$osgiVersion"
```

Comment:
I have seen that "-noimport:=true" is used in the reactor-test module we could use that here too which would make the manifest a little bit cleaner because the exported packages are then not listed in the import-package attribute and there is not really any downside to it (that I know of).

The specification says following (3.6.5):
> An export definition does not imply an automatic import definition. A bundle that exports a package and does not import that package will get that package via its bundle class path. Such an exported only package can be used by other bundles, but the exporting bundle does not accept a substitution for this package from another bundle.

I just think that it should be consistent between all reactor modules. So if you use -noimport for one module it should be used for the others too if not than it should be removed for the reactor-test module (?)

Reply:
please open a separate issue/pr if you find that causes actual problems, I don't want to change anything that isn't strictly necessary and directly related to the new version scheme in this PR so that the commit serves as a reference for other projects (addons notably)

Reply:
The idea with `-noimport:=true` is very good.